spec-smt/benchmark_ssf.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+# Generate benchmarks for the 3SF problem and time CVC5 on them.
+#
+# This script generates benchmarks with increasing number of blocks and
+# checkpoints, and times CVC5 on each of them. The output is written to
+# stdout in CSV format.
+#
+# WARNING!  This simply replaces a few lines in the model file by offsets.
+# BRITTLE!  If the model file has changed, this script will break!
+
+MODEL_FILENAME = 'ssf.smt'
+BENCHMARK_FILENAME = 'benchmark.smt'
+DATATYPE_LINE_OFFSETS = (28, 32)    # (last line to print, last line to exclude)
```

Comment:
From the current file, shouldn't this be `(29,33)` ?

Reply:
Line 29 should already be excluded, it declares the number of hashes.
Line 33 should be included, `declare-datatype Vote` is independent of the size parameters.